### PR TITLE
{tools}[dummy/] vsc-mympirun v4.1.9

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3113.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/e0a1deab2cc43f895c029bcf7c4bb3f3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2014.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/983f75993e3d3b2ac126f2d17e5fdcb4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2599.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1bfe2337a28a673b63b0bc77f858fdba for a full test report.

**Comment by migueldiascosta** [comment]
closing and reopening to trigger travis

File: easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-4.1.9.eb [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        'source_urls': ['https://pypi.python.org/packages/source/v/vsc-install'],
+        'checksums': ['f1d0796aa6201c0d7e1c6bde591153be1b8bde2f04860494272432c1d30f18f5'],
+    }),
+    ('vsc-base', '2.9.2', {
+        'modulename': 'vsc.utils',
+        'source_urls': ['https://pypi.python.org/packages/source/v/vsc-base'],
+        'checksums': ['55803ce00f29bc0eb8b4e37518b21368b4c0f4cdb37c3a168912fc815fd88f15'],
+    }),
+    (name, version, {
+        'modulename': 'vsc',
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-4.1.9.eb]: fixed in 
0556fe3

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Pull request merged**